config/config.go

```diff
@@ -458,6 +458,8 @@ const (
 	RelabelKeep = "keep"
 	// Drops targets for which the input does match the regex.
 	RelabelDrop = "drop"
+	// Sets a label to the modulus of a hash of labels.
+	RelabelHashMod = "hashmod"
```

Comment:
This should be `RelabelHashMod RelabelAction = "hashmod"` so that the type of `RelabelHashMod` is `RelabelAction` and not `string`. Please also fix the types of RelabelKeep and RelabelDrop.